minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/src/main/java/org/apache/nifi/minifi/MiNiFi.java

```diff
@@ -85,7 +85,7 @@ public MiNiFi(final NiFiProperties properties, ClassLoader rootClassLoader)
         // register the shutdown hook
         Runtime.getRuntime().addShutdownHook(new Thread(() -> {
             // shutdown the jetty server
-            shutdownHook(true);
+            shutdownHook(false);
```

Comment:
Curious, what is the reason for this change?

Reply:
I don't believe it should reload at that point (indeed NiFi has it set to false), setting it true causes the close() method to be called twice although I did put defensive code around that in this PR just in case.

Reply:
Thanks for the explanation!